src/webui/components/PackageList/index.js

```diff
@@ -54,9 +48,9 @@ export default class PackageList extends React.Component {
     return (
       <div className="package-list-items">
         <div className={classes.pkgContainer}>
-          {help ? <Help /> : this.renderPackges()}
+          {help ? <Help /> : this.renderPackages()}
```

Comment:
`{help ? <Help /> : this.renderPackages()}` I personally prefer do this logic always on functions, my reason is simple, try to stop execution on dev tools in this line, not easy on JSX sometimes. Any thougths @priscilawebdev ?